File: test/fixtures/output/java/asynchttp/query.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("GET", "https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();

File: src/targets/php/http2/fixtures/query.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('https://httpbin.org/anything');
$request->setRequestMethod('GET');
$request->setQuery(new http\QueryString([
  'foo' => [
    'bar',
    'baz'
  ],
  'baz' => 'abc',
  'key' => 'value'
]));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();